### PR TITLE
Add branching info to contributing

### PR DESCRIPTION
**Pull request by nathangobinet**

### CODE REVIEW

**Review by adrien-nf — APPROVED** [review]
Nice work!

**Review by tylp — APPROVED** [review]

**Review by vbouillennec — APPROVED** [review]

**Review by cyrilgourgouillon — APPROVED** [review]
Nice

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,3 +33,35 @@ Each component consists of four files :
...
+> ***See*** [Git branching workflow](https://www.atlassian.com/fr/git/tutorials/comparing-workflows/gitflow-workflow) article and [Branch naming guidelines](https://gist.github.com/revett/88ee5abf5a9a097b4c88).
+
+In order to allow a better collaboration, the git workflow is structured as follow:
+- `main` branch is the common development branch, features and bugfix should be merged on this branch.
+- `release` branch is used only for stable release, it can be merged only from the `release` branch.
+- `hotfix` branch is used to fix bug in production. Commit must include a tag for the version.
+- Separate branch for development features and bugfix. Must be merged with the `main` branch. Indicate at the begining of the branch name if it is a feature or a bugfix. 
+
+***Specify in the first or last commit of a new branch, the revelant issue(s).***
+
```
Comment (tylp) [inline CONTRIBUTING.md:49]: What the fuck

Comment (vbouillennec) [reply CONTRIBUTING.md:49]: wtf !

**Pull request merged**